package.json

```diff
@@ -31,11 +31,11 @@
   ],
   "main": "./client/out/extension",
   "scripts": {
-    "compile": "tsc -b",
+    "compile": "tsc -b && rollup -c",
     "compile:test": "tsc -b server/src/tests",
     "compile:integration": "tsc -b integration",
     "format": "scripts/format.sh",
-    "watch": "tsc -b -w",
+    "watch": "tsc -b -w; rollup -c -w",
```

Comment:
Maybe use `&` instead of `;` if we want to launch both of these processes, as `tsc -b -w` will block?

Reply:
Good call! This actually broke the vscode dev workflow because rollup was not run.